test/acse-conf/acse-regression.json

```diff
@@ -71,11 +61,6 @@
       "location": "koreacentral"
     },
     {
-      "cluster_definition": "keyvaultcerts/swarm-windows.json",
```

Comment:
windows is part of old header, so keep the test, just make sure locked to older header.